content/app/ui/popups/goto-file-popup.js

```diff
@@ -30,15 +39,30 @@ define("content/app/ui/popups/goto-file-popup", function(require, exports, modul
 		 * Init popup.
 		 */
 		_init : function () {
+
+			// Fuzzy set
+			this._fuzzySet = new FuzzySet();
+			this._entries.forEach(function(entry) {
+				this._fuzzySet.add( entry.toString() );
+			}, this);
+
 			// create input
 			this._input = document.createElement('input');
 			this._input.type = 'text';
 			this._input.className = 'popup-goto-file-input';
 			this.view.appendChild(this._input);
+
 			// create label
 			var label = document.createElement('div');
 			label.className = 'popup-goto-file-label';
 			this.view.appendChild(label);
+
+			// init jquery autocomplete widget
+			$(this._input).autocomplete({
```

Comment:
:+1: 
I've added view property to abstract popup, so that rest of the classes can work normally (PopupManager, GotoLinePopup, etc). We will change them incrementally.